src/chrome/content/rules/libusb.info.xml

```diff
@@ -0,0 +1,5 @@
+<ruleset name="www.libusb.info">
+	<target host="www.libusb.info" />
+	<target host="libusb.info" />
+	<rule from="^http://(www\.)?" to="https://" />
```

Comment:
This syntax is invalid.

Please also add a comment for the non functional hosts `api.libusb.info` and `log.libusb.info` on the top.

Reply:
https://github.com/libusb/libusb/issues/910